plugins/provider-exporters-old.koplugin/main.lua

```diff
@@ -0,0 +1,23 @@
+local Provider = require("provider")
+local WidgetContainer = require("ui/widget/container/widgetcontainer")
+
+-- this is a hack, remove me
+package.path = "plugins/exporter.koplugin/?.lua;" .. package.path
```

Comment:
Not sure how to tackle this.

Is it too bad to include it on our own package.path?

The other way is to move exporter.koplugin/base.lua to frontend/export.lua or something similar.

Reply:
Shouldn't it go at the end if done this way?

Reply:
> Shouldn't it go at the end if done this way?

😄 No idea.

Reply:
I mainly mean in the sense that with a path override you want to put things in front, but we're just adding here rather than overriding.

Reply:
Yay!

I ended up adding it in `setupkoenv.lua`.

Moving the base exporter itself to frontend isn't enough, as exporters might also use one of the templates (markdown, html). Moving everything to base sounds a bit crazy.

Conditionally add the path when the exporter plugin is enabled isn't as option. Providers are independent plugins that can be enabled on their own.